components/definitions/newTab.d.ts

```diff
@@ -184,6 +184,7 @@ declare namespace NewTab {
     optInTotal: boolean
     optInBTCPrice: boolean
     optInMarkets: boolean
+    fetchStatus: string | null
```

Comment:
could we make this a type? perhaps `type CryptoDotComFetchStatus = 'completed' | 'refreshing' | 'pending' | ''`?

Reply:
Absolutely, @ryanml. Would that type simply be defined within the same file?

Reply:
See https://github.com/brave/brave-core/pull/6895/commits/29ac20211016c82c2ba50fad010876ce6015dcfc

I kept `fetchStatus` allowing `null` due to initial state set here https://github.com/brave/brave-core/blob/29ac20211016c82c2ba50fad010876ce6015dcfc/components/brave_new_tab_ui/storage/new_tab_storage.ts#L118

Thoughts? Note, I'm not able to test this due to failing build.